lib/utils/ember.js

```diff
@@ -131,6 +135,10 @@ function isEmberObject(node) {
   );
 }
 
+function isEmberTest(filePath) {
+  return filePath.indexOf('tests') > -1;
```

Comment:
Hmm, I think we might be better served to check for the file path ending with `-test`. This would still allow for co-located tests outside of the `tests/` folder (which is a thing we are generally planning to do...).

Reply:
Good point, I updated the code to match `-test` instead